### PR TITLE
Added Default logic to refs

### PR DESCRIPTION
**Pull request by RichardLitt**
Not sure about the switch statement. There may be a more elegant solution.

Part of #2484

License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

File: core/commands/refs.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ NOTE: List all references recursively by using the flag '-r'.
 		cmds.StringArg("ipfs-path", true, true, "Path to the object(s) to list refs from.").EnableStdin(),
 	},
 	Options: []cmds.Option{
-		cmds.StringOption("format", "Emit edges with given format. Available tokens: <src> <dst> <linkname>."),
+		cmds.StringOption("format", "Emit edges with given format. Available tokens: <src> <dst> <linkname>.").Default("refs"),
```
Comment (Kubuxu) [inline core/commands/refs.go]: This is wrong.

It should be `<dst>`.

Comment (RichardLitt) [inline core/commands/refs.go]: Fixed.

**Comment by Kubuxu** [comment]
LGTM

File: core/commands/refs.go [hunk 2]
```diff
@@ -313,6 +313,8 @@ func (rw *RefWriter) WriteEdge(from, to key.Key, linkname string) error {
 
 	var s string
 	switch {
+	case rw.PrintFmt == "<dst>":
```
Comment (whyrusleeping) [inline core/commands/refs.go]: I don't think this case is actually needed, the next case will do what youre doing here already

Comment (Kubuxu) [inline core/commands/refs.go]: It might be a simple (possibly premature) optimization.

99% of uses of this command will use this format string.

Comment (RichardLitt) [inline core/commands/refs.go]: That was my thinking: is this a premature opt?

Comment (whyrusleeping) [inline core/commands/refs.go]: yeah, i'd call this a premature op. The added time of going through the other case will be dwarfed by the cost of a b58 marshal and api latency

**Comment by RichardLitt** [comment]
Fixed.

**Pull request merged**